install_config/registry/extended_registry_configuration.adoc

```diff
@@ -569,6 +569,14 @@ capability also makes it possible for you to pull images that reside on a
 registry they do not have credentials to access, as long as you have access
 to the image stream tag that references the image.
 
+You must ensure that your registry has appropriate certificatess to trust 
+any external registries you do a pullthrough against. The certificates need to be placed 
+in the *_/etc/pki/tls/certs_* directory on the pod. You can mount the certifiates 
+using a xref:../..//dev_guide/configmaps.adoc#configmaps-creating-from-directories[configuration map] 
```

Comment:
extra / in the file path. Be sure to run asciibinder build to preview and test the links.

Reply:
Thanks. It did work in preview!